xml/System/Environment.xml

```diff
@@ -1573,7 +1553,7 @@ The following example creates environment variables for the <xref:System.Environ
       </ReturnValue>
       <Docs>
         <summary>Gets the newline string defined for this environment.</summary>
-        <value>A string containing "\r\n" for non-Unix platforms, or a string containing "\n" for Unix platforms.</value>
+        <value><code>"\r\n"</code> for non-Unix platforms, or a string containing <code>"\n"</code> for Unix platforms.</value>
```

Comment:
Check this in the preview.

Reply:
https://review.docs.microsoft.com/en-us/dotnet/api/system.environment.newline?view=dotnet-uwp-10.0&branch=pr-en-us-4674#property-value